app/assets/stylesheets/layout.scss

```diff
@@ -37,6 +37,15 @@
   margin-top: 30px;
 }
 
+.mt-45 {
```

Comment:
not a big fan of em? ;-)

Reply:
Just being consistent with what's already there :D